src/commands/utils.js

```diff
@@ -8,16 +8,27 @@
  */
 
 /**
- * Returns the parent table.
+ * Returns the parent table. Returns undefined if position is not inside table.
  *
  * @param {module:engine/model/position~Position} position
  * @returns {module:engine/model/element~Element|module:engine/model/documentfragment~DocumentFragment}
  */
 export function getParentTable( position ) {
```

Comment:
Is this alias really helpful?

Reply:
Not now - it was before, but I'd remove it as the `getParentElement( 'tableCell' )` is more helpfull. I'll ook also if any existing engine method is not better here...